### ISSUE TITLE
Jitsi link don't expire as expected

### ISSUE THREAD

**Issue by DHaussermann** [mattermost/mattermost-plugin-jitsi#84]
When creating a Jitsi link, the post says the post is valid for 30 minutes ex: `Meeting link valid until: Thu Jun 04 2020 07:07:58 GMT-0400 (Eastern Daylight Time)`

But you can go back and join meeting IDs that are several hours or days old. This is a bit confusing and I'm not sure of the expected behavior should be, particularly for Personal meeting types where the meeting ID never changes so the link may always be valid. 

Steps:
- Create a meeting of any naming_scheme 
- Note the time the meeting link is valid for 
- Return 31 minutes later
- Click the link
**Observed** Meeting link is still functional

**Comment by jespino** [comment]
Yes, actually is a tricky thing, probably is not very clear, and probably shouldn't be exposed this way, or at least take an action after that time. The thing is, if you create a meeting with JWT, the JWT have a timeframe where it is valid. The link still work, but the JWT is not accepted. That means, in an environment where you requires to be validated to access a meeting, the link is not going to work, but in an environment where you accept usage without a user, you will be able to enter anyway, but without the data incorporated from mattermost. That means, if you try to access to any of this links after the 30 minutes, the mattermost user data shouldn't be shown in Jitsi.

**Comment by DHaussermann** [comment]
Thanks @jespino a related question: Does Jitsi support some type of webhook event that could update  a post once it's expired? It's a bit strange because it's not clear to me when jitsi meeting actually end. So not sure what other time it would be appropriate for the post to update.

**Comment by jespino** [comment]
Is not supported by jitsi for now, probably we can try to add that feature to the jitsi backend, but currently is not there.

There is an API that is not very usable from go, but I have to investigate a bit more around that.
